packages/automation/.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+  "extends": ["../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
```

Comment:
This file can be removed as the linting will be handled from a root config here
https://github.com/govuk-one-login/govuk-one-login-frontend/pull/5/files#diff-0947e2727d6bad8cd0ac4122f5314bb5b04e337393075bc4b5ef143b17fcbd5b

I'm thinking that maybe the linting pr should be merged before this one, just so we can ensure this all matches up correctly